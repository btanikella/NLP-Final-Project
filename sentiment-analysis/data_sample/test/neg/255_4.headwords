m
admires
reminds
n`t
guess
