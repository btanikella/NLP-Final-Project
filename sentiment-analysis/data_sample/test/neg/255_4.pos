PRP  NN IN CD NNS IN NNP  DT NN NN IN NNP
NN IN PRP VBZ PRP IN VBG CD NNS TO VB CC NN IN PRP VBZ PRP IN VBG RB NN NN
IN JJ NNS PRP VBZ PRP IN DT NN NN NN  DT NN PRP VBD  CC PRP MD VB VB PRP$ NN IN DT JJ NN IN DT NN WP VBZ TO VB NN VBG NNS RB
MD VB PRP VB RBR JJ IN NN VBG TO VB DT NN TO VB IN NN IN DT NN CC VB
UH UH PRP VBP NNP VBZ JJ NN NNS VBP IN RB IN PRP VBD DT DT NN NN VBG $ CD IN CD CD NNP NNS MD VB IN PRP CC IN CD CD NNS MD VB IN PRP  CC PRP VBP TO VB IN DT NN MD VB RB RB VBN IN PRP VBD JJ VBN IN DT NN IN NNP
