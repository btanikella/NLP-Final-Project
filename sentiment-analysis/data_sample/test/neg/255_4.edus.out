Sentence #1 (16 tokens):
I`m in two minds about FOLLOWING , the film debut of Christopher Nolan .
[Text=I CharacterOffsetBegin=4 CharacterOffsetEnd=5 PartOfSpeech=PRP] [Text=` CharacterOffsetBegin=5 CharacterOffsetEnd=6 PartOfSpeech=``] [Text=m CharacterOffsetBegin=6 CharacterOffsetEnd=7 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=8 CharacterOffsetEnd=10 PartOfSpeech=IN] [Text=two CharacterOffsetBegin=11 CharacterOffsetEnd=14 PartOfSpeech=CD] [Text=minds CharacterOffsetBegin=15 CharacterOffsetEnd=20 PartOfSpeech=NNS] [Text=about CharacterOffsetBegin=21 CharacterOffsetEnd=26 PartOfSpeech=IN] [Text=FOLLOWING CharacterOffsetBegin=27 CharacterOffsetEnd=36 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=37 CharacterOffsetEnd=38 PartOfSpeech=,] [Text=the CharacterOffsetBegin=39 CharacterOffsetEnd=42 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=43 CharacterOffsetEnd=47 PartOfSpeech=NN] [Text=debut CharacterOffsetBegin=48 CharacterOffsetEnd=53 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=54 CharacterOffsetEnd=56 PartOfSpeech=IN] [Text=Christopher CharacterOffsetBegin=57 CharacterOffsetEnd=68 PartOfSpeech=NNP] [Text=Nolan CharacterOffsetBegin=69 CharacterOffsetEnd=74 PartOfSpeech=NNP] [Text=. CharacterOffsetBegin=75 CharacterOffsetEnd=76 PartOfSpeech=.] 
root(ROOT-0, m-3)
nsubj(m-3, I-1)
punct(m-3, `-2)
prep(m-3, in-4)
num(minds-6, two-5)
pobj(in-4, minds-6)
prep(minds-6, about-7)
pobj(about-7, FOLLOWING-8)
punct(m-3, ,-9)
det(debut-12, the-10)
nn(debut-12, film-11)
appos(m-3, debut-12)
prep(debut-12, of-13)
nn(Nolan-15, Christopher-14)
pobj(of-13, Nolan-15)
punct(m-3, .-16)

Sentence #2 (23 tokens):
Part of me admires it for costing 6,000 dollars to make but part of me hates it for being too art house .
[Text=Part CharacterOffsetBegin=81 CharacterOffsetEnd=85 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=86 CharacterOffsetEnd=88 PartOfSpeech=IN] [Text=me CharacterOffsetBegin=89 CharacterOffsetEnd=91 PartOfSpeech=PRP] [Text=admires CharacterOffsetBegin=92 CharacterOffsetEnd=99 PartOfSpeech=VBZ] [Text=it CharacterOffsetBegin=100 CharacterOffsetEnd=102 PartOfSpeech=PRP] [Text=for CharacterOffsetBegin=103 CharacterOffsetEnd=106 PartOfSpeech=IN] [Text=costing CharacterOffsetBegin=107 CharacterOffsetEnd=114 PartOfSpeech=VBG] [Text=6,000 CharacterOffsetBegin=115 CharacterOffsetEnd=120 PartOfSpeech=CD] [Text=dollars CharacterOffsetBegin=121 CharacterOffsetEnd=128 PartOfSpeech=NNS] [Text=to CharacterOffsetBegin=129 CharacterOffsetEnd=131 PartOfSpeech=TO] [Text=make CharacterOffsetBegin=132 CharacterOffsetEnd=136 PartOfSpeech=VB] [Text=but CharacterOffsetBegin=137 CharacterOffsetEnd=140 PartOfSpeech=CC] [Text=part CharacterOffsetBegin=141 CharacterOffsetEnd=145 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=146 CharacterOffsetEnd=148 PartOfSpeech=IN] [Text=me CharacterOffsetBegin=149 CharacterOffsetEnd=151 PartOfSpeech=PRP] [Text=hates CharacterOffsetBegin=152 CharacterOffsetEnd=157 PartOfSpeech=VBZ] [Text=it CharacterOffsetBegin=158 CharacterOffsetEnd=160 PartOfSpeech=PRP] [Text=for CharacterOffsetBegin=161 CharacterOffsetEnd=164 PartOfSpeech=IN] [Text=being CharacterOffsetBegin=165 CharacterOffsetEnd=170 PartOfSpeech=VBG] [Text=too CharacterOffsetBegin=171 CharacterOffsetEnd=174 PartOfSpeech=RB] [Text=art CharacterOffsetBegin=175 CharacterOffsetEnd=178 PartOfSpeech=NN] [Text=house CharacterOffsetBegin=179 CharacterOffsetEnd=184 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=185 CharacterOffsetEnd=186 PartOfSpeech=.] 
root(ROOT-0, admires-4)
nsubj(admires-4, Part-1)
prep(Part-1, of-2)
pobj(of-2, me-3)
dobj(admires-4, it-5)
prep(admires-4, for-6)
pcomp(for-6, costing-7)
num(dollars-9, 6,000-8)
dobj(costing-7, dollars-9)
aux(make-11, to-10)
xcomp(costing-7, make-11)
cc(make-11, but-12)
nsubj(hates-16, part-13)
prep(part-13, of-14)
pobj(of-14, me-15)
conj(make-11, hates-16)
dobj(hates-16, it-17)
prep(hates-16, for-18)
cop(house-22, being-19)
advmod(house-22, too-20)
nn(house-22, art-21)
pcomp(for-18, house-22)
punct(admires-4, .-23)

Sentence #3 (40 tokens):
In many ways it reminds me of the cult movie PI , a film I disliked , and I can`t get my head around the central plot of a man who wants to be writer following people around .
[Text=In CharacterOffsetBegin=191 CharacterOffsetEnd=193 PartOfSpeech=IN] [Text=many CharacterOffsetBegin=194 CharacterOffsetEnd=198 PartOfSpeech=JJ] [Text=ways CharacterOffsetBegin=199 CharacterOffsetEnd=203 PartOfSpeech=NNS] [Text=it CharacterOffsetBegin=204 CharacterOffsetEnd=206 PartOfSpeech=PRP] [Text=reminds CharacterOffsetBegin=207 CharacterOffsetEnd=214 PartOfSpeech=VBZ] [Text=me CharacterOffsetBegin=215 CharacterOffsetEnd=217 PartOfSpeech=PRP] [Text=of CharacterOffsetBegin=218 CharacterOffsetEnd=220 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=221 CharacterOffsetEnd=224 PartOfSpeech=DT] [Text=cult CharacterOffsetBegin=225 CharacterOffsetEnd=229 PartOfSpeech=NN] [Text=movie CharacterOffsetBegin=230 CharacterOffsetEnd=235 PartOfSpeech=NN] [Text=PI CharacterOffsetBegin=236 CharacterOffsetEnd=238 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=239 CharacterOffsetEnd=240 PartOfSpeech=,] [Text=a CharacterOffsetBegin=241 CharacterOffsetEnd=242 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=243 CharacterOffsetEnd=247 PartOfSpeech=NN] [Text=I CharacterOffsetBegin=248 CharacterOffsetEnd=249 PartOfSpeech=PRP] [Text=disliked CharacterOffsetBegin=250 CharacterOffsetEnd=258 PartOfSpeech=VBD] [Text=, CharacterOffsetBegin=259 CharacterOffsetEnd=260 PartOfSpeech=,] [Text=and CharacterOffsetBegin=261 CharacterOffsetEnd=264 PartOfSpeech=CC] [Text=I CharacterOffsetBegin=265 CharacterOffsetEnd=266 PartOfSpeech=PRP] [Text=ca CharacterOffsetBegin=267 CharacterOffsetEnd=269 PartOfSpeech=MD] [Text=n`t CharacterOffsetBegin=269 CharacterOffsetEnd=272 PartOfSpeech=VB] [Text=get CharacterOffsetBegin=273 CharacterOffsetEnd=276 PartOfSpeech=VB] [Text=my CharacterOffsetBegin=277 CharacterOffsetEnd=279 PartOfSpeech=PRP$] [Text=head CharacterOffsetBegin=280 CharacterOffsetEnd=284 PartOfSpeech=NN] [Text=around CharacterOffsetBegin=285 CharacterOffsetEnd=291 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=292 CharacterOffsetEnd=295 PartOfSpeech=DT] [Text=central CharacterOffsetBegin=296 CharacterOffsetEnd=303 PartOfSpeech=JJ] [Text=plot CharacterOffsetBegin=304 CharacterOffsetEnd=308 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=309 CharacterOffsetEnd=311 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=312 CharacterOffsetEnd=313 PartOfSpeech=DT] [Text=man CharacterOffsetBegin=314 CharacterOffsetEnd=317 PartOfSpeech=NN] [Text=who CharacterOffsetBegin=318 CharacterOffsetEnd=321 PartOfSpeech=WP] [Text=wants CharacterOffsetBegin=322 CharacterOffsetEnd=327 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=328 CharacterOffsetEnd=330 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=331 CharacterOffsetEnd=333 PartOfSpeech=VB] [Text=writer CharacterOffsetBegin=334 CharacterOffsetEnd=340 PartOfSpeech=NN] [Text=following CharacterOffsetBegin=341 CharacterOffsetEnd=350 PartOfSpeech=VBG] [Text=people CharacterOffsetBegin=351 CharacterOffsetEnd=357 PartOfSpeech=NNS] [Text=around CharacterOffsetBegin=358 CharacterOffsetEnd=364 PartOfSpeech=RB] [Text=. CharacterOffsetBegin=365 CharacterOffsetEnd=366 PartOfSpeech=.] 
root(ROOT-0, reminds-5)
prep(reminds-5, In-1)
amod(ways-3, many-2)
pobj(In-1, ways-3)
nsubj(reminds-5, it-4)
dobj(reminds-5, me-6)
prep(reminds-5, of-7)
det(PI-11, the-8)
nn(PI-11, cult-9)
nn(PI-11, movie-10)
pobj(of-7, PI-11)
punct(PI-11, ,-12)
det(film-14, a-13)
appos(PI-11, film-14)
nsubj(disliked-16, I-15)
rcmod(film-14, disliked-16)
punct(PI-11, ,-17)
cc(reminds-5, and-18)
conj(reminds-5, I-19)
dep(I-19, ca-20)
dep(ca-20, n`t-21)
dep(n`t-21, get-22)
poss(head-24, my-23)
dobj(get-22, head-24)
prep(get-22, around-25)
det(plot-28, the-26)
amod(plot-28, central-27)
pobj(around-25, plot-28)
prep(plot-28, of-29)
det(man-31, a-30)
pobj(of-29, man-31)
nsubj(wants-33, who-32)
rcmod(man-31, wants-33)
aux(writer-36, to-34)
cop(writer-36, be-35)
xcomp(wants-33, writer-36)
prep(writer-36, following-37)
pobj(following-37, people-38)
advmod(people-38, around-39)
punct(reminds-5, .-40)

Sentence #4 (23 tokens):
Wouldn`t it be more logical for someone wanting to be a writer to sit in front of a keyboard and write ?
[Text=Would CharacterOffsetBegin=371 CharacterOffsetEnd=376 PartOfSpeech=MD] [Text=n`t CharacterOffsetBegin=376 CharacterOffsetEnd=379 PartOfSpeech=VB] [Text=it CharacterOffsetBegin=380 CharacterOffsetEnd=382 PartOfSpeech=PRP] [Text=be CharacterOffsetBegin=383 CharacterOffsetEnd=385 PartOfSpeech=VB] [Text=more CharacterOffsetBegin=386 CharacterOffsetEnd=390 PartOfSpeech=RBR] [Text=logical CharacterOffsetBegin=391 CharacterOffsetEnd=398 PartOfSpeech=JJ] [Text=for CharacterOffsetBegin=399 CharacterOffsetEnd=402 PartOfSpeech=IN] [Text=someone CharacterOffsetBegin=403 CharacterOffsetEnd=410 PartOfSpeech=NN] [Text=wanting CharacterOffsetBegin=411 CharacterOffsetEnd=418 PartOfSpeech=VBG] [Text=to CharacterOffsetBegin=419 CharacterOffsetEnd=421 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=422 CharacterOffsetEnd=424 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=425 CharacterOffsetEnd=426 PartOfSpeech=DT] [Text=writer CharacterOffsetBegin=427 CharacterOffsetEnd=433 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=434 CharacterOffsetEnd=436 PartOfSpeech=TO] [Text=sit CharacterOffsetBegin=437 CharacterOffsetEnd=440 PartOfSpeech=VB] [Text=in CharacterOffsetBegin=441 CharacterOffsetEnd=443 PartOfSpeech=IN] [Text=front CharacterOffsetBegin=444 CharacterOffsetEnd=449 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=450 CharacterOffsetEnd=452 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=453 CharacterOffsetEnd=454 PartOfSpeech=DT] [Text=keyboard CharacterOffsetBegin=455 CharacterOffsetEnd=463 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=464 CharacterOffsetEnd=467 PartOfSpeech=CC] [Text=write CharacterOffsetBegin=468 CharacterOffsetEnd=473 PartOfSpeech=VB] [Text=? CharacterOffsetBegin=474 CharacterOffsetEnd=475 PartOfSpeech=.] 
root(ROOT-0, n`t-2)
aux(n`t-2, Would-1)
nsubj(logical-6, it-3)
cop(logical-6, be-4)
advmod(logical-6, more-5)
ccomp(n`t-2, logical-6)
prep(logical-6, for-7)
pobj(for-7, someone-8)
partmod(someone-8, wanting-9)
aux(writer-13, to-10)
cop(writer-13, be-11)
det(writer-13, a-12)
xcomp(wanting-9, writer-13)
aux(sit-15, to-14)
dep(writer-13, sit-15)
prep(sit-15, in-16)
pobj(in-16, front-17)
prep(front-17, of-18)
det(keyboard-20, a-19)
pobj(of-18, keyboard-20)
cc(sit-15, and-21)
conj(sit-15, write-22)
punct(n`t-2, ?-23)

Sentence #5 (65 tokens):
Oh well I guess FOLLOWING gives indie film makers hope that just because they made a no budget movie costing $6,000 over two thousand IMDB members will vote for it and over five hundred members will comment on it , but you have to wonder if this movie would be so well regarded if it wasn`t made by the director of MEMENTO ?
[Text=Oh CharacterOffsetBegin=480 CharacterOffsetEnd=482 PartOfSpeech=UH] [Text=well CharacterOffsetBegin=483 CharacterOffsetEnd=487 PartOfSpeech=UH] [Text=I CharacterOffsetBegin=488 CharacterOffsetEnd=489 PartOfSpeech=PRP] [Text=guess CharacterOffsetBegin=490 CharacterOffsetEnd=495 PartOfSpeech=VBP] [Text=FOLLOWING CharacterOffsetBegin=496 CharacterOffsetEnd=505 PartOfSpeech=NNP] [Text=gives CharacterOffsetBegin=506 CharacterOffsetEnd=511 PartOfSpeech=VBZ] [Text=indie CharacterOffsetBegin=512 CharacterOffsetEnd=517 PartOfSpeech=JJ] [Text=film CharacterOffsetBegin=518 CharacterOffsetEnd=522 PartOfSpeech=NN] [Text=makers CharacterOffsetBegin=523 CharacterOffsetEnd=529 PartOfSpeech=NNS] [Text=hope CharacterOffsetBegin=530 CharacterOffsetEnd=534 PartOfSpeech=VBP] [Text=that CharacterOffsetBegin=535 CharacterOffsetEnd=539 PartOfSpeech=IN] [Text=just CharacterOffsetBegin=540 CharacterOffsetEnd=544 PartOfSpeech=RB] [Text=because CharacterOffsetBegin=545 CharacterOffsetEnd=552 PartOfSpeech=IN] [Text=they CharacterOffsetBegin=553 CharacterOffsetEnd=557 PartOfSpeech=PRP] [Text=made CharacterOffsetBegin=558 CharacterOffsetEnd=562 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=563 CharacterOffsetEnd=564 PartOfSpeech=DT] [Text=no CharacterOffsetBegin=565 CharacterOffsetEnd=567 PartOfSpeech=DT] [Text=budget CharacterOffsetBegin=568 CharacterOffsetEnd=574 PartOfSpeech=NN] [Text=movie CharacterOffsetBegin=575 CharacterOffsetEnd=580 PartOfSpeech=NN] [Text=costing CharacterOffsetBegin=581 CharacterOffsetEnd=588 PartOfSpeech=VBG] [Text=$ CharacterOffsetBegin=589 CharacterOffsetEnd=590 PartOfSpeech=$] [Text=6,000 CharacterOffsetBegin=590 CharacterOffsetEnd=595 PartOfSpeech=CD] [Text=over CharacterOffsetBegin=596 CharacterOffsetEnd=600 PartOfSpeech=IN] [Text=two CharacterOffsetBegin=601 CharacterOffsetEnd=604 PartOfSpeech=CD] [Text=thousand CharacterOffsetBegin=605 CharacterOffsetEnd=613 PartOfSpeech=CD] [Text=IMDB CharacterOffsetBegin=614 CharacterOffsetEnd=618 PartOfSpeech=NNP] [Text=members CharacterOffsetBegin=619 CharacterOffsetEnd=626 PartOfSpeech=NNS] [Text=will CharacterOffsetBegin=627 CharacterOffsetEnd=631 PartOfSpeech=MD] [Text=vote CharacterOffsetBegin=632 CharacterOffsetEnd=636 PartOfSpeech=VB] [Text=for CharacterOffsetBegin=637 CharacterOffsetEnd=640 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=641 CharacterOffsetEnd=643 PartOfSpeech=PRP] [Text=and CharacterOffsetBegin=644 CharacterOffsetEnd=647 PartOfSpeech=CC] [Text=over CharacterOffsetBegin=648 CharacterOffsetEnd=652 PartOfSpeech=IN] [Text=five CharacterOffsetBegin=653 CharacterOffsetEnd=657 PartOfSpeech=CD] [Text=hundred CharacterOffsetBegin=658 CharacterOffsetEnd=665 PartOfSpeech=CD] [Text=members CharacterOffsetBegin=666 CharacterOffsetEnd=673 PartOfSpeech=NNS] [Text=will CharacterOffsetBegin=674 CharacterOffsetEnd=678 PartOfSpeech=MD] [Text=comment CharacterOffsetBegin=679 CharacterOffsetEnd=686 PartOfSpeech=VB] [Text=on CharacterOffsetBegin=687 CharacterOffsetEnd=689 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=690 CharacterOffsetEnd=692 PartOfSpeech=PRP] [Text=, CharacterOffsetBegin=693 CharacterOffsetEnd=694 PartOfSpeech=,] [Text=but CharacterOffsetBegin=695 CharacterOffsetEnd=698 PartOfSpeech=CC] [Text=you CharacterOffsetBegin=699 CharacterOffsetEnd=702 PartOfSpeech=PRP] [Text=have CharacterOffsetBegin=703 CharacterOffsetEnd=707 PartOfSpeech=VBP] [Text=to CharacterOffsetBegin=708 CharacterOffsetEnd=710 PartOfSpeech=TO] [Text=wonder CharacterOffsetBegin=711 CharacterOffsetEnd=717 PartOfSpeech=VB] [Text=if CharacterOffsetBegin=718 CharacterOffsetEnd=720 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=721 CharacterOffsetEnd=725 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=726 CharacterOffsetEnd=731 PartOfSpeech=NN] [Text=would CharacterOffsetBegin=732 CharacterOffsetEnd=737 PartOfSpeech=MD] [Text=be CharacterOffsetBegin=738 CharacterOffsetEnd=740 PartOfSpeech=VB] [Text=so CharacterOffsetBegin=741 CharacterOffsetEnd=743 PartOfSpeech=RB] [Text=well CharacterOffsetBegin=744 CharacterOffsetEnd=748 PartOfSpeech=RB] [Text=regarded CharacterOffsetBegin=749 CharacterOffsetEnd=757 PartOfSpeech=VBN] [Text=if CharacterOffsetBegin=758 CharacterOffsetEnd=760 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=761 CharacterOffsetEnd=763 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=764 CharacterOffsetEnd=767 PartOfSpeech=VBD] [Text=n`t CharacterOffsetBegin=767 CharacterOffsetEnd=770 PartOfSpeech=JJ] [Text=made CharacterOffsetBegin=771 CharacterOffsetEnd=775 PartOfSpeech=VBN] [Text=by CharacterOffsetBegin=776 CharacterOffsetEnd=778 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=779 CharacterOffsetEnd=782 PartOfSpeech=DT] [Text=director CharacterOffsetBegin=783 CharacterOffsetEnd=791 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=792 CharacterOffsetEnd=794 PartOfSpeech=IN] [Text=MEMENTO CharacterOffsetBegin=795 CharacterOffsetEnd=802 PartOfSpeech=NNP] [Text=? CharacterOffsetBegin=803 CharacterOffsetEnd=804 PartOfSpeech=.] 
root(ROOT-0, guess-4)
advmod(guess-4, Oh-1)
advmod(guess-4, well-2)
nsubj(guess-4, I-3)
nsubj(gives-6, FOLLOWING-5)
ccomp(guess-4, gives-6)
amod(makers-9, indie-7)
nn(makers-9, film-8)
dobj(gives-6, makers-9)
dobj(gives-6, hope-10)
mark(vote-29, that-11)
advmod(made-15, just-12)
mark(made-15, because-13)
nsubj(made-15, they-14)
advcl(vote-29, made-15)
det(movie-19, a-16)
det(movie-19, no-17)
nn(movie-19, budget-18)
dobj(made-15, movie-19)
partmod(movie-19, costing-20)
dobj(costing-20, $-21)
num($-21, 6,000-22)
quantmod(thousand-25, over-23)
number(thousand-25, two-24)
num(members-27, thousand-25)
nn(members-27, IMDB-26)
nsubj(vote-29, members-27)
aux(vote-29, will-28)
ccomp(hope-10, vote-29)
prep(vote-29, for-30)
pobj(for-30, it-31)
cc(vote-29, and-32)
prep(comment-38, over-33)
number(hundred-35, five-34)
num(members-36, hundred-35)
pobj(over-33, members-36)
aux(comment-38, will-37)
conj(vote-29, comment-38)
prep(comment-38, on-39)
pobj(on-39, it-40)
punct(guess-4, ,-41)
cc(guess-4, but-42)
nsubj(have-44, you-43)
conj(guess-4, have-44)
aux(wonder-46, to-45)
xcomp(have-44, wonder-46)
mark(regarded-54, if-47)
det(movie-49, this-48)
nsubjpass(regarded-54, movie-49)
aux(regarded-54, would-50)
auxpass(regarded-54, be-51)
advmod(well-53, so-52)
advmod(regarded-54, well-53)
advcl(wonder-46, regarded-54)
mark(n`t-58, if-55)
nsubj(n`t-58, it-56)
cop(n`t-58, was-57)
advcl(regarded-54, n`t-58)
dep(n`t-58, made-59)
prep(made-59, by-60)
det(director-62, the-61)
pobj(by-60, director-62)
prep(director-62, of-63)
pobj(of-63, MEMENTO-64)
punct(guess-4, ?-65)

